src/components/OurWork.astro

```diff
@@ -1,209 +1,201 @@
-                            </div> 
-                )   
-        }
-    </div>
+<h2>
+	<a href="/work">
+		Our Work
+	</a>
+</h2>
+<div class="our-work">
+	{
+		data.sort((a, b) => a.work - b.work)
+			.map((post) => { return {...post, work: post.work-1 } })
+			.map((post) => (
+				<div 
+					id={`work-${post.work}`} 
+					class={`work work-${(post.work == 0) ? 'selected': 'unselected'}`}
+				>
+					<h3>
+						{post.title}
+					</h3>
+					<p>
+						{post.astro.source}
+						<a 
+							class="read-more"
+							href={`${post.url}`}
+							target="_blank"
+						>
+							Read More
+						</a>
+					</p>
+				</div> 
+			))
+	}
 </div>
 
-<CtaButtom link="/work" idName="bottom-cta" > See All Our Work </CtaButtom>
+<CtaButtom link="/work" idName="bottom-cta" >
+	See All Our Work
+</CtaButtom>
 
 
 <script>
-        let current = 0;
-        let total = document.getElementsByClassName("work").length;
-
-        function changeWork(change = 1) {
-            let cur = document.getElementById(`work-${current}`);
-            cur.classList.remove("work-selected")
-            cur.classList.add("work-unselected");
-            current = (current + change) % total;
-            let next = document.getElementById(`work-${current}`);
-            next.classList.remove("work-unselected")
-            next.classList.add("work-selected")
-        }
-
-        setInterval(changeWork, 7000);
+	let current = 0;
+	let total = document.getElementsByClassName("work").length;
+
+	function changeWork(change = 1) {
+		let cur = document.getElementById(`work-${current}`);
+		cur.classList.remove("work-selected")
+		cur.classList.add("work-unselected");
+		current = (current + change) % total;
+		let next = document.getElementById(`work-${current}`);
+		next.classList.remove("work-unselected")
+		next.classList.add("work-selected")
+	}
+
+	setInterval(changeWork, 7000);
 </script>   
 
 <style>
-
 @keyframes quote-in {
-    0% {
-        color: var(--tertiary);
-        opacity: 0;
-        -webkit-transform: translateX(40px);
-        transform: translateX(40px);
-    }
+	0% {
+		color: var(--tertiary);
+		opacity: 0;
+		transform: translateX(40px);
+	}
 
-    90% {
-        opacity: 1;
-    }
+	90% {
+		opacity: 1;
+	}
 
-    100% {
-        opacity: 1;
-        -webkit-transform: translateX(0px);
-        transform: translateX(0px);
-    }
+	100% {
+		opacity: 1;
+		transform: translateX(0px);
+	}
 }
 
 @keyframes quote-out {
-    0% {
-        opacity: 1;
-        -webkit-transform: translateX(0px);
-        transform: translateX(0px);
-    }
+	0% {
+		opacity: 1;
+		transform: translateX(0px);
+	}
 
-    90% {
-        color: var(--secondary);
-        opacity: 0;
-    }
+	90% {
+		color: var(--secondary);
+		opacity: 0;
+	}
 
-    100% {
-        opacity: 0;
-        -webkit-transform: translateX(-40px);
-        transform: translateX(-40px);
-    }
+	100% {
+		opacity: 0;
+		transform: translateX(-40px);
+	}
 }
 
 h2 {
-    font-size: 30pt;
```

Comment:
The only thing worse that `px` font units is `pt` font units. I've replaced them throughout with `rem` units roughly equal (roughly because you can't tell the difference and I don't want `1.777778rem`)

Reply:
lol